cli/cmd/component.go

```diff
@@ -76,6 +78,15 @@ var (
 		Args:    cobra.ExactArgs(1),
 		RunE:    runComponentsDelete,
 	}
+
+	// componentsDevModeCmd represents the dev sub-command inside the components command
+	componentsDevModeCmd = &cobra.Command{
+		Use:    "dev <component>",
+		Hidden: true,
+		Short:  "Enter development mode of a new or existing component",
+		Args:   cobra.ExactArgs(1),
+		RunE:   runComponentsDevMode,
```

Comment:
nit: I know this is hidden, but a 'long' description with more details on cmd usage would be nice.